hammer.sh

```diff
@@ -552,6 +584,8 @@ function install_deps_AppImageKit()
     cd $DEPS_BUILD/AppImageKit
     curl -OL https://raw.github.com/probonopd/AppImageKit/master/AppImageAssistant.AppDir/package
     curl -OL https://raw.github.com/probonopd/AppImageKit/master/AppImageAssistant.AppDir/xdgappdir.py
+    #AppImageKit has trouble finding Ember's icon. Let's make this simpler.
+    sed -i 's/icon_name, 48, None, format/icon_name, 48/' xdgappdir.py
```

Comment:
Is this because it's placed in an incorrect location to start with? Or is the issue with AppImageKit?